src/Messages/InvalidKeyMessages/TSInvalidIdentityKeyErrorMessage.m

```diff
@@ -29,6 +25,11 @@ - (NSString *)theirSignalId
     return nil;
 }
 
+- (BOOL)isDynamicInteraction
+{
+    return YES;
```

Comment:
I haven't really seen how this is used yet - presumably it'll come up in a forthcoming Signal-iOS pull request, but Identity key errors seem very different from contact offers / blocking offers.

My immediate concern is that the others are treated as sort of disposable because they can be recreated, but the identity key error message can not. What benefit/functionality do we get by marking them as dynamic?